ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/runs/$runIndex/index.tsx

```diff
@@ -61,7 +61,30 @@ const RunComponent = () => {
       }),
   });
 
-  const downloadZipFile = async ({
+  const downloadLogs = async (runId: number) => {
+    try {
+      // TODO: use a readymade query from TanStack query client for this?
```

Comment:
Nit: Start capitalized after a colon 😉

Reply:
Really, in English? In Finnish, it's lower-case.

Reply:
TanStack Query should in fact not be used for this, so the whole comment should be removed, which resolves this. 😎

Reply:
> Really, in English?

In American English, yes, esp. if the following sentence is a full sentence.